test/stats/stats_aggregate_test.cpp

```diff
@@ -0,0 +1,237 @@
+// Copyright (c) 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Tests for unique type declaration rules validator.
+
+#include <string>
+
+#include "source/spirv_stats.h"
+#include "test_fixture.h"
+#include "unit_spirv.h"
+
+namespace {
+
+using libspirv::SpirvStats;
+using spvtest::ScopedContext;
+
+void AggregateStats(const std::string& code, SpirvStats* stats,
```

Comment:
It's very confusing that this helper function has the same name as the thing you are testing.